package.json

```
@@ -0,0 +1,66 @@
+{
+  "name": "hometry-fe",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@emotion/react": "^11.13.3",
+    "@emotion/styled": "^11.13.0",
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^16.18.108",
+    "@types/react-dom": "^18.3.0",
+    "date-fns": "^4.1.0",
+    "react": "^18.3.1",
+    "react-day-picker": "^9.1.1",
+    "react-dom": "^18.3.1",
+    "react-router": "^6.26.2",
+    "react-router-dom": "^6.26.2",
+    "react-scripts": "5.0.1",
+    "typescript": "^4.9.5",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "lint": "eslint './src/**/*.{ts,tsx,js,jsx}'",
+    "lint:fix": "eslint --fix './src/**/*.{ts,tsx,js,jsx}'",
+    "prettier": "prettier --write --config ./.prettierrc './src/**/*.{ts,tsx}'"
+  },
+  "eslintConfig": {
+    "extends": [
+      "react-app",
+      "react-app/jest"
+    ]
+  },
```

`.eslintrc`로 파일 분리해서 eslint 관련 설정 관리하기 때문에 package.json에서 eslintConfig는 제거해도 될 거 같아용

```suggestion
  }
```